src/main/java/bio/terra/service/snapshotbuilder/query/tables/Table.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.service.snapshotbuilder.query.tables;
```

Comment:
I wonder if this should be `table` instead? Package names are usually singular 
```suggestion
package bio.terra.service.snapshotbuilder.query.table;
```